registration/forms.py

```diff
@@ -0,0 +1,10 @@
+'''importing django forms'''
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+class SignupForm(UserCreationForm):
```

Comment:
I will prefer you to create your own signup form and handle user creation yourself to understand how user creation works.

Reply:
Okay